newrelic.js

```diff
@@ -43,4 +43,18 @@ exports.config = {
       "response.headers.x*",
     ],
   },
+  rules: {
+    name: [
+      { pattern: "/api/lanes/*", name: "Nextjs/GET//api/lanes/[slug]/" },
+      { pattern: "/api/lanes", name: "Nextjs/GET//api/lanes" },
+      { pattern: "/api/homepage", name: "Nextjs/GET//api/homepage" },
+      { pattern: "/api/feedback", name: "Nextjs/POST//api/feedback" },
+      { pattern: "/api/featuredItem", name: "Nextjs/GET//api/featuredItem" },
+      { pattern: "/divisions/*", name: "/Nextjs/GET/divisions/[slug]/" },
+      { pattern: "/divisions", name: "Nextjs/GET//divisions" },
+      { pattern: "/about", name: "Nextjs/GET//about" },
+      { pattern: "/404", name: "Nextjs/GET//404" },
+      { pattern: "/", name: "Nextjs/GET//" },
+    ],
+  },
```

Comment:
I think this approach is cleaner than adding `newrelic.setTransactionName('Nextjs/GET//about');` to the server component but maybe that's just me. 

Regardless, we'll need to remember to add a line of code somewhere for New Relic to pick up any new pages. It might be worth adding the phase 3 and phase 4 routes to this file now. 

Regardless, this approach is acceptable to me.

Reply:
I don't want to add any routes which are not live in prod just yet. There's little risk of doing so but don't want anyone to get the wrong idea of seeing those events captures and thinking that it's somehow live to patrons.